documentation/modules/about-cold-warm-migration.adoc

```diff
@@ -47,3 +47,51 @@ The precopy stage runs until the cutover stage is started manually or is schedul
 The VMs are shut down during the cutover stage and the remaining data is migrated. Data stored in RAM is not migrated.
 
 You can start the cutover stage manually by using the {project-short} console or you can schedule a cutover time in the `Migration` manifest.
+
+[id="warm-migration-versus-cold-migration"_{context}]
+== Cold migration vs. warm migration: Which is right for your migration?
+
+Both warm migration and cold migration have advantages and limitations. In general, warm migrations can take longer than cold migrations, but warm migrations involve less downtime than cold migrations.
```

Comment:
Can we convert the data of this sentence into a table like the following


|  | Duration | VM Downtime |
|---|---|---|
| Cold Migration | Minimal | High |
| Warm migration | Longer and variable | Minimal |